community/kernel/src/main/java/org/neo4j/kernel/impl/store/CountsComputer.java

```diff
@@ -58,33 +61,43 @@ public CountsComputer( NeoStores stores )
     }
 
     public CountsComputer( long lastCommittedTransactionId, NodeStore nodes, RelationshipStore relationships,
-            int highLabelId,
-            int highRelationshipTypeId )
+            int highLabelId, int highRelationshipTypeId )
+    {
+        this( lastCommittedTransactionId, nodes, relationships, highLabelId, highRelationshipTypeId,
+                new SilentMigrationProgressMonitor().NO_OP_SECTION );
```

Comment:
There is no reason to create new instance if SilentMigrationProgressMonitor here since NO_OP_SECTION is static